diff --git NuttX/nuttx/arch/arm/src/stm32f7/chip/stm32f76xx77xx_pinmap.h NuttX/nuttx/arch/arm/src/stm32f7/chip/stm32f76xx77xx_pinmap.h
index ef5fec6..f94ab14 100644
--- NuttX/nuttx/arch/arm/src/stm32f7/chip/stm32f76xx77xx_pinmap.h
+++ NuttX/nuttx/arch/arm/src/stm32f7/chip/stm32f76xx77xx_pinmap.h
@@ -996,7 +996,7 @@
 #define GPIO_SPI2_SCK_3       (GPIO_ALT|GPIO_AF5|GPIO_SPEED_50MHz|GPIO_PORTB|GPIO_PIN13)
 #define GPIO_SPI2_SCK_4       (GPIO_ALT|GPIO_AF5|GPIO_SPEED_50MHz|GPIO_PORTD|GPIO_PIN3)
 #define GPIO_SPI2_SCK_5       (GPIO_ALT|GPIO_AF5|GPIO_SPEED_50MHz|GPIO_PORTI|GPIO_PIN1)
-#define GPIO_SPI2_SCK_6       (GPIO_ALT|GPIO_AF5|GPIO_SPEED_50MHz|GPIO_PORTIA|GPIO_PIN12)
+#define GPIO_SPI2_SCK_6       (GPIO_ALT|GPIO_AF5|GPIO_SPEED_50MHz|GPIO_PORTI|GPIO_PIN12)
 
 #define GPIO_SPI3_MISO_1      (GPIO_ALT|GPIO_AF6|GPIO_SPEED_50MHz|GPIO_PORTB|GPIO_PIN4)
 #define GPIO_SPI3_MISO_2      (GPIO_ALT|GPIO_AF6|GPIO_SPEED_50MHz|GPIO_PORTC|GPIO_PIN11)
